packages-web/fieldset/src/Fieldset.editorPreview.tsx

```diff
@@ -0,0 +1,19 @@
+import { createElement, ReactElement } from "react";
+import { parseStyle } from "@widgets-resources/piw-utils";
+
+import { Fieldset } from "./components/Fieldset";
+import { FieldsetPreviewProps } from "../typings/FieldsetProps";
+
+export const preview = (props: FieldsetPreviewProps): ReactElement => {
+    const { class: className, legend } = props;
+    const style = parseStyle(props.style);
+    const ContentRenderer = props.content.renderer;
```

Comment:
Maybe `Content` instead of ` ContentRenderer` is more descriptive

Reply:
I think `ContentRenderer` is better here, because it isn't the content itself. The component takes a `div` element where it injects the actual content. Otherwise it wouldn't make sense why we are passing a `div` element as a child to `ContentRenderer`. I used the same naming conventions as in the docs.